fastlane/lib/fastlane/actions/environment_variable.rb

```diff
@@ -0,0 +1,73 @@
+module Fastlane
```

Comment:
new action? why?

Reply:
Great question, if you want to set environment variable in Fastfile.swift, the only way to get _fastlane_ to actually use them, is to pass them along. That's because they operate as 2 separate processes and so the ruby process doesn't inherit the environment.

Reply:
Something for that technical blog post ;)

Reply:
Can we add that justification either to a comment or the description of this action? Just so that folks know when to use this vs using ENV[].